src/modules/flow/ipm/ipm.json

```diff
@@ -0,0 +1,475 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "ipm",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "Apache-2.0",
+    "version": "1"
+  },
+  "types": [
+    {
+      "category": "output/ipm",
+      "description": "Inter-Processor Messaging boolean writer. Writes a boolean packet with a given ID to a different core.",
+      "methods": {
+        "open": "boolean_writer_open"
+      },
+      "name": "ipm/boolean-writer",
+      "options": {
+        "members": [
+          {
+            "data_type": "int",
```

Comment:
you need an `uint32_t`, but it's not available. However I believe we should add it to the generator (and schema) instead of propagating the casts in the code. This looks like a reasonable type to add.